keyboards/crkbd/keymaps/shinze/keymap.c

```diff
@@ -0,0 +1,269 @@
+  ),
+
+  [_RAISE] = LAYOUT(
+  //,------+--------+--------+--------+--------+--------.   ,-------+--------+---------+--------+--------+---------.
+    BP_DLR , BP_DQOT, BP_LGIL, BP_RGIL, BP_LPRN, BP_RPRN,    BP_AT  , BP_PLUS, BP_MINUS, BP_SLSH, BP_ASTR, BP_EQUAL,
+  //,------+--------+--------+--------+--------+--------.   ,-------+--------+---------+--------+--------+---------.
+    _______, _______ ,_______ , _______, KC_UP , BP_SPC ,    _______, _______ , _______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+   ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, KC_LEFT, KC_DOWN,KC_RIGHT,    KC_ESC , _______ ,_______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+   ,-------+--------+---------+--------+--------+---------.
+                               BP_RALT, BP_RGUI, _______,    KC_RCTRL,BP_ENT , BSPC
+  ),
+
+  [_LOWER] = LAYOUT(
+  //,------+--------+--------+--------+--------+--------.    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,    _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, SAVE   , COPY   , PASTE  , CTAB   ,    _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, BP_RALT, ALTSFT,     _______, _______ ,_______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+    ,-------+--------+---------+--------+--------+---------.
+                               _______, _______, _______,    _______, _______, _______
+  ),
+
+  [_ADJUST] = LAYOUT(
+  //,------+--------+--------+--------+--------+--------.   ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,   _______, _______, _______  , _______, _______ ,  KC_RST,
+  //,------+--------+--------+--------+--------+--------.   ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,    _______, _______, _______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+   ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,    _______, _______ ,_______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+   ,-------+--------+---------+--------+--------+---------.
+                               _______, _______, _______,    _______, _______, _______
+  )
+
+};
+
+int RGB_current_mode;
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
This function is depreciated, and has a core function that supersedes it.  

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer